DEVELOPMENT.md

```diff
@@ -22,11 +20,33 @@ GO_BINARY_PATH=go1.11.X
 or whichever binary you want to use with athens
 ```
 
+# Run the Proxy
+After you've set up your dependencies, the `buffalo` CLI makes it easy to launch the proxy: 
+
+```console
+cd cmd/proxy
+buffalo dev
+```
+
+If you don't have Buffalo installed, you can just use the `go` command directly as such: 
+
+```
+cd cmd/proxy
+go build
+./proxy
+```
+
+After the server starts, you'll see some console output like:
+
+```console
+Starting application at 127.0.0.1:3000
+```
+
 ### Dependencies
 
 # Services that Athens Needs
 
-Athens relies on several services (i.e. databases, etc...) to function properly. We use [Docker](http://docker.com/) images to configure and run those services.
+Athens relies on several services (i.e. databases, etc...) to function properly. We use [Docker](http://docker.com/) images to configure and run those services. **However, Athens does not require any storage dependencies by default**. The default storage is in memory, you can opt-in to using the `fs` which would also require no dependencies. But if you'd like to test out Athens against a real storage backend (such as MongoDB, Minio, S3 etc), continue reading this section:
```

Comment:
I thought the default storage is disk?

Reply:
@arschles we had a discussion back in the day and ended up with Memory

Reply:
oh wait - are you talking about https://github.com/gomods/athens/blob/master/config.dev.toml#L61 here? if so, ignore me and sorry to confuse this 😄